### ISSUE TITLE
Plans to extend mne-bids also to mri-data?

### ISSUE THREAD

**Issue by JohannesWiesner** [mne-tools/mne-bids#617]
Hi, first of all, thank you for creating this useful repository. Without knowing of mne-bids I created a repository called [nisupply](https://github.com/JohannesWiesner/nisupply/blob/master/nisupply.py) which follows a very similar logic. I linked your repo in my README file. Are there plans to extend mne-bids also to mri data types in the future? I am using [CAT12](http://www.neuro.uni-jena.de/cat/) to preprocess my structural MRI images. CAT12 is not yet bids-conform, so it breaks the BIDS-structure and thus I had to write my own code (there are also other issues as described in this [GitHub issue](https://github.com/bids-standard/bids-specification/issues/654) which make my dataset non-BIDS-conform). Long story short, I needed some generic helper functions to overcome these issues. In case you have plans to extend the functionality of mne-bids also to other data types I would be happy to contribute.

**Comment by jasmainak** [comment]
what kind of generic helper functions are you looking for?

I don't think going into MRI world is in the plans per se but no harm in adding some generic functions if they are useful. Have you looked at `pybids` for MRI?

**Comment by eort** [comment]
@JohannesWiesner in case it is an option for you (and `mne-bids` is not going to move into the MRI world), I found it rather straightforward to use existing tools (e.g. [heudiconv](https://github.com/nipy/heudiconv)) to make my MRI data BIDS-formatted and then have a little script to integrate it with the BIDS dataset produced by `mne-bids`

**Comment by JohannesWiesner** [comment]
@eort I actually really don't know why I posted this question back in November, of course MNE-BIDS is not going to move into the MRI world, stupid me. I guess I was new to BIDS and didn't see the whole picture. This issue can be closed. I had trouble with combining pybids with my data set, that was supposed to be BIDS-conform but couldn't be read by pybids (because at that time pybids didn't support multi-echo tags, don't know if this was solved in the meantime). 

Anyways, I got along by creating the [nisupply module](https://github.com/JohannesWiesner/nisupply) and/or used the  `nipype` file selection tools (see [related projects](https://github.com/JohannesWiesner/nisupply#similar-projects))
